docs/topics/recipes.md

```diff
@@ -233,7 +233,7 @@ the cell.
 Here is how to achieve this in PhpSpreadsheet:
 
 ```php
-$spreadsheet->getActiveSheet()->getCell('A1')->setValue("hello\nworld");
+$spreadsheet->getActiveSheet()->getCell('A1')->setValue("hello\nworld");//pay attention to the double quotes
```

Comment:
If you feel that it's important to remind developers that double quotes are needed, then it might also be worthwhile explaining why they are necessary